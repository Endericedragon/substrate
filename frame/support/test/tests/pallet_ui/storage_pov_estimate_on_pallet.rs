#[frame_support::pallet]
mod pallet {
	use frame_support::pallet_prelude::*;

	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	#[pallet::pov_estimate = MaxEncodedLen]
	pub struct Pallet<T>(_);

	#[pallet::error]
	pub enum Error<T> {}

	#[pallet::storage]
	type Foo<T: Config> = StorageValue<_, u8>;
}

fn main() {
}
