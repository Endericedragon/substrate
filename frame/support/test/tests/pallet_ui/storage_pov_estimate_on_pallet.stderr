error: expected one of: `generate_store`, `without_storage_info`, `storage_version`
 --> tests/pallet_ui/storage_pov_estimate_on_pallet.rs:9:12
  |
9 |     #[pallet::pov_estimate = MaxEncodedLen]
  |               ^^^^^^^^^^^^
